importer/import.py

```diff
@@ -34,11 +34,13 @@ def pg_execute(sql, params=None):
     )
 """
 
-def import_cosmogony_to_pg(cosmogony_path):
-    pg_execute("""
-        DROP TABLE IF EXISTS public.zones;
 
-        CREATE TABLE IF NOT EXISTS public.zones(
+def _import_cosmogony_to_pg(cosmogony_path):
```

Comment:
The table name needs to be changed in each `CREATE INDEX` too. By default, it tries to alter `public.zones`